user_sync/credentials.py

```diff
@@ -21,7 +21,7 @@
 yaml.indent(mapping=4, sequence=4, offset=2)
 
 
-# from ruamel.yaml.scalarstring import PreservedScalarString as pss
+from ruamel.yaml.scalarstring import PreservedScalarString as pss
```

Comment:
Can you move this above with the rest  andclean up the imports?  (ctrl+alt+o)  Also delete comment